src/styles/sidebar/components/top-bar.scss

```diff
@@ -77,37 +69,16 @@
   &:hover {
     color: $gray-dark;
   }
-  &--active {
+
+  &.is-active {
     color: $gray-dark;
   }
-}
 
-// Button which indicates that other users have made or edited annotations
```

Comment:
At one point in the past I presume there was some sort of visible update count, but not in my memory. These styles were really not being used. Most don't make sense for the current appearance of the SVG icon. Shrug?

Reply:
👍